framework/addons/data/addons.json

```diff
@@ -67,15 +67,15 @@
             },
             "View / download file versions": {
                 "status": "partial",
-                "text": "figshare content can be viewed and downloaded via OSF provided it is \"published\" on figshare."
+                "text": "figshare files can be viewed/downloaded via OSF, but version history is not supported."
             },
             "Add / update files": {
                 "status": "partial",
                 "text": "Files can be added but not updated."
             },
             "Delete files": {
-                "status": "none",
-                "text": "figshare files cannot be deleted via OSF."
+                "status": "partial",
```

Comment:
Do you mean in Erin's PR? (#6523)  That does not mention delete capabilities, only metadata.  Figshare doesn't seem to be mentioned anywhere else in the API.